rgs/rgs-server/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rust:latest as builder
+
+WORKDIR /usr/src
+RUN git clone --depth 1 https://github.com/lightningdevkit/rapid-gossip-sync-server/
+WORKDIR /usr/src/rapid-gossip-sync-server
+RUN git reset --hard f0ef17314840b4e30d08511dfc7773b4ffc8d343
+COPY ./config.rs /usr/src/rapid-gossip-sync-server/src/config.rs
```

Comment:
I agree with you. The methodology is inconsistent. I'll try to also use `sed` for `config.rs`. I avoided doing it because the changes are a little more complex.

Reply:
Done in bdefcdb :)